tools/integration_tests/write_large_files/seq_write_large_file_test.go

```diff
@@ -39,13 +41,35 @@ func TestWriteLargeFileSequentially(t *testing.T) {
 		t.Errorf("Error in writing file: %v", err)
 	}
 
+	// Download the file from a bucket in which we write the content.
```

Comment:
Create a method to compare two files and use that at appropriate places. You can refer to optimise it - https://github.com/udhos/equalfile/blob/90a7461a2a226fa62eca6e350b5207ae9c78fff8/equalfile.go#L105